app/views/users/admin/activity.scala.html

```diff
@@ -0,0 +1,52 @@
+@import db.ModelService
+@import db.impl.access.UserBase
+@import java.sql.Timestamp
+@import ore.OreConfig
+@import models.project.{Project, Version}
+@import models.admin.Review
+@import util.StringUtils._
+@import security.NonceFilter._
+@import models.user.User
+
+@import models.project.Flag
+
+@import java.time.Instant
+@(user: User, activities: Seq[Object])(implicit messages: Messages, request: Request[_], service: ModelService, config: OreConfig, users: UserBase)
+
+@bootstrap.layout(messages("activity.title") + user.username) {
+
+    <div class="container" style="margin-top: 90px">
+        <div class="row">
+            <div class="col-md-12  header-flags">
+                <div class="clearfix">
+                    <h1 class="pull-left">@messages("activity.title") <i>@user.username</i></h1>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <table class="table table-condensed setting-no-border table-review-log">
+                    <tbody>
+                        @if(activities.isEmpty) {
+                            <tr><td>No activities founds</td></tr>
+                        }
+                        @for(activity <- activities) {
+                            @if(activity.isInstanceOf[Review]) {
+                                <tr>
+                                    <td>Review approved:</td>
+                                    <td>@prettifyDateAndTime(activity.asInstanceOf[Review].endedAt.getOrElse(Timestamp.from(Instant.EPOCH)))</td>
+                                </tr>
+                            }
+                            @if(activity.isInstanceOf[Flag]) {
+                                <tr>
+                                    <td>Flag resolved:</td>
```

Comment:
Have this link to whatever project got flagged (or if there's a "flag" page link to that)